composer.json

```diff
@@ -27,6 +27,10 @@
     "drupal/focal_point": "^1.2",
     "drupal/google_tag": "^1.2",
     "drupal/honeypot": "^1.30",
+    "drupal/layout_builder_modal": "1.x-dev",
```

Comment:
Dev is currently the same as last stable version so we better have stable version 1.1 here.